### PR TITLE
Add Endchan Rules

### PR DESCRIPTION
**Pull request by gitgud-software**
This commit adds HTTPS Everywhere rules for the [Endchan](https://infinow.net/) imageboard.

### CODE REVIEW

File: src/chrome/content/rules/Endchan.xyz.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="Endchan.xyz">
+	<target host="endchan.xyz" />
+	<target host="www.endchan.xyz" />
```
Comment (semenko) [inline src/chrome/content/rules/Endchan.xyz.xml]: Looks like this doesn't work (cert name doesn't include www)

**Comment by gitgud-software** [comment]
This adjustment should resolve the matter for now, I'll add the "www" resolution back in when the SSL certificate properly covers the subdomain.

**Comment by semenko** [comment]
Looks like you removed the wrong www

**Comment by gitgud-software** [comment]
Alright, now we're good.

**Comment by semenko** [comment]
Awesome -- thanks! :100:

**Pull request merged**